apix-rest-v1/src/main/java/eu/xenit/apix/rest/v1/ApixSwaggerDescription.java

```diff
@@ -12,12 +13,12 @@
         info = @Info(
                 description = "This is the swagger specification for Api-X REST API" +
                         "\n\n" +
-                        "Examples can be found at: https://xenitsupport.jira.com/wiki/display/APIX/REST+API+Examples",
+                        "Examples can be found at: https://docs.xenit.eu/alfred-api",
```

Comment:
:+1:

Reply:
Remember to replace that page with a link to docs.xenit.eu when the website has been deployed.

Reply:
That link should go to the front page of the documentation site. For other products its a landing for the version. you can compare with https://docs.xenit.eu/alfred-inflow

Reply:
Maybe I was unclear: with page I meant our internal Confluence page that should be cleaned. But there is already a Jira ticket for that so that's good.